nbs/075_models.RNNAttention.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            Args:
</pre><p>You should replace the line that explain what this model is. It's certainly not TST. I don't know if you are basing this model on any particular paper, or it's just an implementation you have created based on your experience. Can you please comment on it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timeseriesAI/tsai/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Sorry for the mistake. The model is apparently based on the TST model you implement. I created it for my own need. And I also see someone filed issue #595. Therefore I made this PR. I'll edit it.